### PR TITLE
Seltest suffix

### PR DESCRIPTION
**Pull request by zoltan-l**
BFW-1428

### CODE REVIEW

File: src/common/selftest/selftest_MINI.cpp [hunk 1]
```diff
@@ -278,25 +278,16 @@ void CSelftest::log_open() {
         serial_otp = 0;
     }
     char serial[32] = "unknown";
-    const char *suffix = "";
-    switch (m_Mask) {
-    case stmFans:
+    const char *suffix = "_no_suffix";
```
Comment (DRracer) [inline src/common/selftest/selftest_MINI.cpp:281]: why is this initialization needed?

File: src/common/selftest/selftest_MINI.cpp [hunk 2]
```diff
@@ -278,25 +278,16 @@ void CSelftest::log_open() {
...
+    if (m_Mask & stmFans)
         suffix = "_fans";
-        break;
-    case stmXYAxis:
-    case stmHome_XYAxis:
+    if (m_Mask & stmXYAxis)
```
Comment (DRracer) [inline src/common/selftest/selftest_MINI.cpp:284]: can we have multiple suffices? This code looks like we could (no "else if"), but the suffix pointer is always changed hard

**Review by DRracer — CHANGES_REQUESTED** [review]
Please address my comments, I'm not sure about the intent and side effects.
Also, a PR consisting of Fix, format, format doesn't look good, please improve the commits by rebasing interactively

File: src/common/selftest/selftest_MINI.cpp (hunk 1 above)
Comment (zoltan-l) [reply src/common/selftest/selftest_MINI.cpp:281]: for identification and testing purposes- when the further conditions do not meet, empty char does not provide any suffix

File: src/common/selftest/selftest_MINI.cpp (hunk 2 above)
Comment (zoltan-l) [reply src/common/selftest/selftest_MINI.cpp:284]: else if will be used

File: src/common/selftest/selftest_MINI.cpp (hunk 1 above)
Comment (DRracer) [reply src/common/selftest/selftest_MINI.cpp:281]: why do you need any suffix at all in such case?

**Review by DRracer — APPROVED** [review]

**Pull request merged**